plugins/inputs/opcua/README.md

```diff
@@ -276,6 +279,20 @@ This example group configuration has three groups with two nodes each:
     ]
 ```
 
+### Server Certificate Trust
+
+When connecting to OPC UA servers with self-signed certificates using
+secure modes (Sign or SignAndEncrypt), you need to explicitly trust the
+server's certificate. Use the `remote_certificate` option to specify the
+path to the server's certificate file.
+
+#### Obtaining the Server Certificate
+
```

Comment:
Maybe remove this caption and put it in the text, but that's just a matter of taste not a must.